music_assistant/server/providers/radiobrowser/__init__.py

```diff
@@ -203,10 +197,10 @@ async def get_tag_names(self, limit: int, offset: int):
             tag_names.append(tag.name.lower())
         return tag_names
 
-    async def get_country_codes(self, limit: int, offset: int):
+    async def get_country_codes(self):
         """Get a list of country names."""
         countries = await self.radios.countries(
-            order=Order.NAME, hide_broken=True, limit=limit, offset=offset
```

Comment:
why did you remove the paging ? Is it broken in the library ?

Reply:
Unfortunately the countries method doesn't have a filter so I thought to leave the paging by slicing, but happy to remove it and get it all in once. Does the method need to add a cache decorator or it pass that over from the main method ?

Reply:
See my reply abovve - it doesn't need a filter, we just want all the items in chunks